schedule-replicas-by-topology-labels.md

```diff
@@ -14,9 +14,156 @@ To improve the high availability and disaster recovery capability of TiDB cluste
 
 To make this mechanism effective, you need to properly configure TiKV and PD so that the topology information of the cluster, especially the TiKV location information, is reported to PD during deployment. Before you begin, see [Deploy TiDB Using TiUP](/production-deployment-using-tiup.md) first.
 
-## Configure `labels` based on the cluster topology
+## Configure `labels` for TiKV, TiFlash, and TiDB
 
-### Configure `labels` for TiKV and TiFlash
+You can configure `labels` for TiKV, TiFlash, and TiDB based on the cluster topology.
+
+### Configure a cluster using TiUP (recommended)
```

Comment:
Tip for reviewers: In the original content, this section is in L130. This PR adds info about TiDB node configuration to this section, and moves this section to the beginning of this doc as this method is recommended.